lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -16,12 +16,16 @@
 from id3c.cli.redcap import CachedProject, is_complete
 from id3c.db.session import DatabaseSession
 from id3c.db.datatypes import Json
+from id3c.cli.command.geocode import pickled_cache
 from . import etl
 
 
 LOG = logging.getLogger(__name__)
 
 
+CACHE_FILE = 'cache.pickle'
```

Comment:
I don't think we should hardcode a cache name like this, particularly with a name that doesn't give any hint as to what uses it or what it contains. The `id3c geocode` command, for instance, explicitly parameterizes the cache file as an option.

Reply:
This does not need to block deploy, but should be addressed later.